### PR TITLE
Implementation of application

### PR DESCRIPTION
**Pull request by Gabbendorf**

### CODE REVIEW

File: src/main/java/Counter.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    public Counter(AppLabel label) {
+        this.label = label;
+    }
+
+    public void updateValue() {
+        value += 1;
+        label.setText(Integer.toString(value));
+    }
+
+    public int getValue() {
```
Comment (RabeaGleissner) [inline src/main/java/Counter.java:15]: Is this method used?

File: src/main/java/Counter.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
+public class Counter {
+
+    private final AppLabel label;
+    private int value;
+
+    public Counter(AppLabel label) {
+        this.label = label;
+    }
+
+    public void updateValue() {
```
Comment (RabeaGleissner) [inline src/main/java/Counter.java:10]: Do you want to try writing a test for this? that value gets increased by one when the method is called?

Comment (Gabbendorf) [reply src/main/java/Counter.java:10]: I made a first attempt but the test failed, not sure why. I'll try again now

File: src/main/java/Counter.java (hunk 1 above)
Comment (Gabbendorf) [reply src/main/java/Counter.java:15]: Not yet, I added it for a test which haven't included in the commit, but apparently I forgot to remove the method before committing

**Pull request merged**